src/test/java/org/folio/rest/api/InventoryViewTest.java

```diff
@@ -108,6 +108,13 @@ public void shouldReturnInstanceEvenIfNoHoldings() {
     }
   }
 
+  /**
+   * nod without barcode to allow multiple items
+   */
+  static JsonObject nod(UUID holdingsRecordId) {
```

Comment:
Given the intention is this item should not have a barcode, I suggest this be described in the method name rather than a comment (meaning a consumer does not need to check the comments / implementation).

```suggestion
  static JsonObject nodWithNoBarcode(UUID holdingsRecordId) {
```